src/services/measure/measure.js

```diff
@@ -0,0 +1,270 @@
+goog.provide('ngeo.interaction');
+goog.provide('ngeo.interaction.Measure');
+
+goog.require('goog.events');
+goog.require('ol.DrawEvent');
+goog.require('ol.Feature');
+goog.require('ol.FeatureOverlay');
+goog.require('ol.MapBrowserEvent');
+goog.require('ol.Observable');
+goog.require('ol.Overlay');
+goog.require('ol.interaction.Pointer');
+goog.require('ol.style.Style');
+
+
+
+/**
+ * @classdesc
+ * Interaction that allows measuring (length, area, ...).
+ *
+ * @constructor
+ * @extends {ol.interaction.Pointer}
+ */
+ngeo.interaction.Measure = function() {
+
+
+  /**
+   * The help tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.helpTooltipElement_;
+
+
+  /**
+   * Overlay to show the help messages.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.helpTooltip_;
+
+
+  /**
+   * The measure tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.measureTooltipElement_;
+
+
+  /**
+   * Overlay to show the measurement.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.measureTooltip_;
+
+
+  /**
+   * Pointer move listener key
+   * @type {goog.events.Key}
+   * @private
+   */
+  this.pointerMoveHandlerKey_;
+
+
+  /**
+   * The sketch feature.
+   * @type {ol.Feature}
+   */
+  this.sketchFeature;
+
+  /**
+   * The draw interaction to be used.
+   * @type {ol.interaction.Draw|ngeo.interaction.DrawAzimut}
+   */
+  this.drawInteraction = null;
+
+  goog.base(this);
+
+
+  /**
+   * The draw overlay
+   * @type {ol.FeatureOverlay}
+   */
+  this.overlay = new ol.FeatureOverlay({
+    style: [
+      new ol.style.Style({
+        fill: new ol.style.Fill({
+          color: 'rgba(255, 255, 255, 0.2)'
+        })
+      }),
+      new ol.style.Style({
+        stroke: new ol.style.Stroke({
+          color: 'white',
+          width: 5
+        })
+      }),
+      new ol.style.Style({
+        stroke: new ol.style.Stroke({
+          color: '#ffcc33',
+          width: 3
+        })
+      })
+    ]
+  });
+};
+goog.inherits(ngeo.interaction.Measure, ol.interaction.Pointer);
+
+
+/**
+ * @inheritDoc
+ */
+ngeo.interaction.Measure.prototype.setMap = function(map) {
+  goog.base(this, 'setMap', map);
+  this.overlay.setMap(map);
+
+  map.addInteraction(this.drawInteraction);
```

Comment:
This should be done automatically.
https://github.com/openlayers/ol3/blob/bfcceac68f321a308979f897e2d51b68b289dd58/src/ol/map.js#L435